Automattic/woocommerce-connect-client
# Fix error field path parsing.

Fixes #365.

To test:
* Cause any validation error for any field, make sure error behavior is unchanged
* Edit a servive adjustment value in a Canada Post instance to be a non-number, like `a` (edit in DB)
* Reload settings page
* Verify that the service group with the errant value is expanded, and the adjustment input decorated with an error
